CODEOWNERS

```diff
@@ -26,8 +26,24 @@
 /bgp/ @cilium/sig-bgp
 /cmd/ @cilium/cli
 /clustermesh/ @cilium/sig-clustermesh
-/connectivity/ @cilium/cli
+/connectivity/ @cilium/ci-structure
```

Comment:
@cilium/ci-structure

Reply:
Reading the description of the @cilium/ci-structure team responsibilities:

https://github.com/cilium/cilium/blob/8639179adb95951286a7d5789e21d5c5f47e8cfc/CODEOWNERS#L36-L39

this doesn't sound like the most appropriate team to solely own all the connectivity tests. There's certainly a testing infrastructure aspect to the connectivity tests, but ideally we'd also want feature owners to review respective test (e.g. `connectivity/check/ipsec.go` which should likely be owned by the @cilium/ipsec team).

How about further refining code owners for connectivity tests in that direction? I think @cilium/ci-structure could then own the "connectivity test infrastructure" while the feature teams own the respective checks.

Reply:
Sorry, I only realized now that the diff also includes changes in that direction already. How about additionally also covering files in `connectivity/check` where appropriate?

Reply:
From a quick look. There might be more.

connectivity/check/ipcache.go @cilium/ipcache and/or @cilium/sig-datapath
connectivity/check/metrics*.go @cilium/metrics
connectivity/check/policy.go @cilium/sig-policy

I'd maybe also files dealing with k8s objects (e.g. connectivity/check/deployment.go) to @cilium/sig-k8s.

Reply:
Good point, I'll go through those and add any missing.

Reply:
I added what you suggested Tobias, but decided against sig-k8s because I thought it didn't make sense for this team to review changes to K8s resources in abstract because in the end those resources will get used in the testing infra. So I think it makes more sense for ci-structure to own the full picture. WDYT?

Reply:
Thanks. Agree, makes sense for ci-structure to keep owning the k8s resource creation.